site/docs/mark/bar.md

```diff
@@ -39,6 +38,16 @@ If we map a different discrete field to the `y` channel, we can produce a horizo
 
 <span class="vl-example" data-name="bar_aggregate"></span>
 
+While the `bar` mark typically encodes x and y channels with a pair of discrete and continuous fields, it can also be used with continuous fields on both x and y channels.
```

Comment:
uses the x and y channels to encode a pair of discrete and continuous fields (reverse the direction of encode)